File: rb/test/exists_test.rb
# ParkingStgallen SDK exists test

require "minitest/autorun"
require_relative "../ParkingStgallen_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = ParkingStgallenSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end

File: lua/test/exists_test.lua
-- ParkingStgallen SDK exists test

local sdk = require("parking-stgallen_sdk")

describe("ParkingStgallenSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
